test/functional/ui/screen_basic_spec.lua

```diff
@@ -614,7 +614,8 @@ local function screen_tests(linegrid)
     it('messages from the same Ex command as resize are visible #22225', function()
       feed(':set columns=20 | call<CR>')
       screen:expect([[
-                            |*9
+                            |
+        {0:~                   }|*8
```

Comment:
This looks like a bugfix as a side-effect of these changes. I don't see why the `~` would not have been included before.